indexing-hadoop/src/main/java/org/apache/druid/indexer/JobHelper.java

```diff
@@ -315,11 +315,21 @@ public static void injectDruidProperties(Configuration configuration, List<Strin
     String mapJavaOpts = StringUtils.nullToEmptyNonDruidDataString(configuration.get(MRJobConfig.MAP_JAVA_OPTS));
     String reduceJavaOpts = StringUtils.nullToEmptyNonDruidDataString(configuration.get(MRJobConfig.REDUCE_JAVA_OPTS));
 
-    for (String propName : System.getProperties().stringPropertyNames()) {
```

Comment:
nit: Do we want to prevent `System.getProperties()` from being used in this project?

Reply:
I think it's fine to not have the exclusion for now, the properties handling stuff is pretty localized in `JobHelper` and I'm not 100% sure that there won't be a legitimate need to call `System.getProperties()` ever in the Hadoop module